packages/ag-charts-community/src/chart/series/seriesAreaManager.ts

```diff
@@ -135,6 +135,10 @@ export class SeriesAreaManager extends BaseManager {
     public readonly focusIndicator?: FocusIndicator;
     private readonly swapChain: FocusSwapChain;
 
+    get bbox() {
+        return BBox.merge([this.seriesRect ?? BBox.zero]);
```

Comment:
```suggestion
        return (this.seriesRect ?? BBox.zero).clone();
```
nit: Why merge on a single item?